### DISCUSSION TITLE
Duplicate string in DNS check error message for custom domains

### DISCUSSION THREAD

**Discussion by vringar** [community/community#17544] [Other Feature Feedback, Questions, & Ideas]
When you have your domain improperly set up you get the following error message

> www.example.com is improperly configured
Your site's DNS settings are using a custom subdomain, www.example.com, that's set up as an A record. We recommend you change this to a CNAME record pointing at [YOUR USERNAME].github.io. For more information, see [Learn more](https://help.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidARecordError). We recommend you change this to a CNAME record pointing to vringar.github.io.

I would assume that you should only get one message either

> We recommend you change this to a CNAME record pointing at [YOUR USERNAME].github.io.

or

> We recommend you change this to a CNAME record pointing to vringar.github.io.

Idk if this is the best way to raise this issue. Please let me know the proper process otherwise.

**Comment by Jok3r42O** [comment]
I think we recommend you change this to a CNAME record pointing to vringar.github.io.

**Reply by vringar** [reply]
Yeah but the message is there twice, once with the formatted string and once with the "template". :sweat_smile:
